js/build.js

```diff
@@ -0,0 +1,51 @@
+Fliplet.Widget.instance({
+  name: 'slide',
+  childOf: ['slider'],
+  data: {
+    slideIndex: 0,
+    formCount: 0,
+    sliderRequiredForms: []
+  },
+  render: {
+    class: 'swiper-slide',
+    template: '<div data-view="content"></div>',
+    ready: async function() {
+      const thisSlide = this;
```

Comment:
You've improved the naming in the [Slider Container](https://github.com/Fliplet/fliplet-widget-slider-container/pull/3/files#diff-a6a69823860696351be715a6f34f14d877f3d832c332b625e19cf4382b03d75aR20) but not for this widget